queries.sql

```diff
@@ -1,12 +1,54 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * FROM animals WHERE name LIKE '%mon';
+
 SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;
+
 SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
+
 SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu'); 
+
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
 SELECT * FROM animals WHERE neutered = true; 
+
 SELECT * FROM animals WHERE name NOT IN ('Gabumon'); 
+
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] Again good job adding the query, however, as mentioned in the requirement please verify if the change was made.